test/functional/apps/discover/_data_grid_doc_table.ts

```diff
@@ -38,10 +38,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await kibanaServer.uiSettings.replace({});
     });
 
-    it('should show the first 12 rows by default', async function () {
+    it('should show the first 11 rows by default', async function () {
       // with the default range the number of hits is ~14000
       const rows = await dataGrid.getDocTableRows();
-      expect(rows.length).to.be(12);
+      expect(rows.length).to.be(11);
```

Comment:
Increased size of EuiPagination slightly reduces the visible area, causing 1 fewer virtualized item.